packages/ra-ui-materialui/src/input/PasswordInput.stories.tsx

```diff
@@ -0,0 +1,80 @@
+import * as React from 'react';
+import polyglotI18nProvider from 'ra-i18n-polyglot';
+import englishMessages from 'ra-language-english';
+
+import { AdminContext } from '../AdminContext';
+import { Create, Edit } from '../detail';
+import { SimpleForm } from '../form';
+import { FormInspector } from './common';
+import { PasswordInput } from './PasswordInput';
+import { Resource } from 'ra-core';
+import { Admin } from 'react-admin';
+import { TextInput } from './TextInput';
+import { BooksCreate } from './common/BooksCreate';
+import { BooksList } from './common/BooksList';
+import { dataProvider } from './common/dataProvider';
+import { history } from './common/history';
+
+export default { title: 'ra-ui-materialui/input/PasswordInput' };
+
+export const Basic = () => (
+    <Wrapper>
+        <PasswordInput source="published" />
```

Comment:
<!-- Polish comments are like a suggestion, where there is nothing necessarily wrong with the relevant content, there's just some ways to immediately improve the quality.  -->
**polish:** The `source‏` should rather be something like `"password"` IMO :grin: 

```suggestion
        <PasswordInput source="password" />
```

Same applies to the whole file